src/test.py

```diff
@@ -22,35 +25,39 @@ def test_rand_overlap():
 
     return True
 
+
 def test_piece_overlap():
     any_x = 1
     any_y = 1
 
     piece = Piece(any_x, any_y)
     return piece.overlaps(any_x, any_y)
```

Comment:
A test for state machine would be nice,eg 

Not overlapping -move-> overlapping/not overlapping